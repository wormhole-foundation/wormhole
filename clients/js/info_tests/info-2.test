info registrations mainnet ethereum TokenBridge -v
