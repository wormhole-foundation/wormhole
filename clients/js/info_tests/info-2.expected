Mismatches found on  Ethereum Mainnet!
{
  Mantle: 'Expected null , found 0x00000000000000000000000024850c6f61c438823f01b7a3bf2b89b72174fa9d'
}
